src/Security/Security/ChangeLog.md

```diff
@@ -18,9 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Added new cmdlet: 'Get-AzSecurityTopology'
+* Added new cmdlet: 'Get-AzAllowedConnections'
 
-## Version 0.7.9
+## Version 0.8.0
```

Comment:
please do not modify the module version here, Azure-PowerShell team will bump it during each release